src/UserEditor.js

```diff
@@ -19,13 +19,13 @@ export class UserEditor extends React.Component {
     handleChange = (key) => async(text) => {
         let {email, updateData} = this.props;
 
-        let response = await userInfo({
+        let response = await getUserData({
             email,
             [key]: text
         });
 
         updateData({
-            userInfo: {...response}
+            userData: {...response}
```

Comment:
why do you need destructurig here?

Reply:
in order to update user information that is used in other components

Reply:
do you mean unnecessary destructuring?

I agree, she is superfluous